### PR TITLE
[WIP] chore: Bump package versions used in installer

### PR DESCRIPTION
**Pull request by jgwerner**
- Update package dependencies used in the installer
- Sync package dependencies used in tests

### CODE REVIEW

File: tests/test_conda.py [hunk 1]
```diff
@@ -13,13 +13,13 @@ def prefix():
     """
     Provide a temporary directory with a conda environment
     """
-    miniconda_version = '4.5.4'
+    miniconda_version = '4.7.10'
     miniconda_installer_md5 = "a946ea1d0c4a642ddf0c3a26a18bb16d"
```
Comment (yuvipanda) [inline tests/test_conda.py]: The md5 hash needs to be updated here as well.

**Comment by jgwerner** [comment]
@yuvipanda thanks, a silly overlook on my part, thanks for helping me get acquainted with this repo. 

there seems to be some overlap with #457, do you want me to send a PR to minrk's fork/branch instead?  in any case i'll test with another branch later to not get in the way of your orgs build limits.

**Comment by jgwerner** [comment]
@yuvipanda do you think we should activate different versions of python with [conda activations](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment)?

Or perhaps just default to py37 #433.

**Comment by jgwerner** [comment]
Closing in favor of linked PRs.

**Pull request closed without merging**